code/modules/goonhub/api/routes/rounds/EndGameRound.dm

```diff
@@ -0,0 +1,9 @@
+
+/// PUT /rounds/end/{gameRound}
+/// End a game round.
+/datum/apiRoute/rounds/end
+	method = RUSTG_HTTP_METHOD_PUT
+	path = "/rounds/end"
+	routeParams = list("gameRound")
+	body = list("crashed")
```

Comment:
no, this isn't just a string body. it's a full json with a type, do this properly.